static/consts.ts

```diff
@@ -0,0 +1,83 @@
+export let messages = [
```

Comment:
```suggestion
export const messages = [
```
Ни на что технически не влияет, кроме ощущения безопасности)

Reply:
Поправил